.github/PULL_REQUEST_TEMPLATE/release_pull_request_template.md

```diff
@@ -0,0 +1,67 @@
+## Description
+
+This PR introduces a new <!-- insert new version here --> release in tezos-packaging.
+
+<!-- Depending on the type of the release some of the points below can be ommited.
+
+E.g. in case we introduce a new changes to our native packages, it's not necessary to bump
+used Tezos sources, create a new release in this repository and update brew formulas.
+
+Note that opam-repository is usually updated with some delay, so CI may fail
+due to lack of updated opam Tezos packages. In such case feel free to comment
+out native-packaging-related CI steps from the pipeline until the opam-repository
+is updated.
+-->
+
+<!-- List issues which are going to be resolved in the new packages from this
+release. Write 'None' if there are no related issues.
+
+For example:
+Follows #202, #210
```

Comment:
It says to list issues that are going to be resolved by the (eventual) release, but I don't think "Follows" represents resolving well enough :thinking: Since AFAIU it _is_ possible to make a release all in one PR, I'd say it should probably still default to "Resolves", and this comment should explain that if something is missing e.g. because of the opam, the line should be changed to "Follows #".

Reply:
The problem is that "Resolves" will close the mentioned issues once the PR is merged. However, some of the issues should be considered resolved only once updated native packages are published